src/isIsogram.test.js

```diff
@@ -6,4 +6,22 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for an empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for an isogram string', () => {
+    expect(isIsogram('roksa')).toBe(true);
```

Comment:
It would be beneficial to test with a more complex isogram, such as 'subdermatoglyphic', to ensure the function can handle larger strings.